test/viewmodels/checkAnswers/sections/destination/DestinationConsigneeDetailsSummarySpec.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers.sections.destination
+
+import base.SpecBase
+import fixtures.messages.sections.destination.DestinationConsigneeDetailsMessages
+import models.CheckMode
+import org.scalatest.matchers.must.Matchers
+import pages.sections.destination.DestinationConsigneeDetailsPage
+import play.api.i18n.Messages
+import play.api.test.FakeRequest
+import uk.gov.hmrc.govukfrontend.views.Aliases.Value
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+class DestinationConsigneeDetailsSummarySpec extends SpecBase with Matchers {
+
+  "DestinationConsigneeDetailsSummary" - {
+
+    lazy val app = applicationBuilder().build()
+
+    Seq(DestinationConsigneeDetailsMessages.English, DestinationConsigneeDetailsMessages.Welsh).foreach { messagesForLanguage =>
+
+      s"when being rendered in lang code of '${messagesForLanguage.lang.code}'" - {
+
+        implicit lazy val msgs: Messages = messages(app, messagesForLanguage.lang)
+
+        "when there's no answer" - {
+
+          "must output no row" in {
```

Comment:
When there is no answer I think this should ideally output 1 row and should say `not provided`, just in case a user manages to go directly to the CYA page without having started this section yet. It was done like this on the guarantor section. This doesn't need to be addressed in this ticket as it may get done in the CYA ticket for this section.

Reply:
That will be handled in the CYA task as that will have it in the copydeck. This was put in as a placeholder for then

Reply:
There is also a scenario which it will not be needed in the check answers page so will leave for the CYA ticket to handle